RecoParticleFlow/PFProducer/src/PFElectronAlgo.cc

```diff
@@ -280,16 +282,8 @@ bool PFElectronAlgo::SetLinks(const reco::PFBlockRef&  blockRef,
 		  break;
 		}
 	      }
-	      
-	      if((Algo == reco::TrackBase::undefAlgorithm ||
-	          Algo == reco::TrackBase::ctf ||
-	          Algo == reco::TrackBase::duplicateMerge ||
-	          Algo == reco::TrackBase::cosmics ||
-	          Algo == reco::TrackBase::initialStep ||
-	          Algo == reco::TrackBase::lowPtTripletStep ||
-	          Algo == reco::TrackBase::pixelPairStep ||
-	          Algo == reco::TrackBase::detachedTripletStep ||
-	          Algo == reco::TrackBase::mixedTripletStep)
+      
+	      if(isGoodTrack
```

Comment:
isGoodTrack includes jetcore and muon iterations in addition to the previously checked cases: is this OK?